atlasclient/utils.py

```diff
@@ -14,29 +14,29 @@
 
 """Utilities for the Python client for the MongoDB Atlas API."""
 
-import logging
-
-from http.client import HTTPConnection
+import json
 
 
 class JSONObject(dict):
-    """Dictionary with dot-notation read access."""
+    """Dictionary object with dot-notation read access."""
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Is this needed?

Reply:
Nope, not needed. Removed in the next commit.